provider_proxy.tf

```diff
@@ -0,0 +1,3 @@
+provider "aws" {
+  alias = "remote"
```

Comment:
where is the region for this provider configured?

Reply:
corresponding provider need to be created in source terraform using this module, and proxy provider need to be setup in the module, more details here
https://www.terraform.io/docs/language/modules/develop/providers.html#proxy-configuration-blocks